gemrb/core/GUI/GameControl.cpp

```diff
@@ -2209,7 +2209,18 @@ bool GameControl::OnMouseUp(const MouseEvent& me, unsigned short Mod)
 		
 		if (overContainer || overDoor || (overInfoPoint && overInfoPoint->Type==ST_TRAVEL && target_mode == TARGET_MODE_NONE)) {
 			// move to the object before trying to interact with it
-			CommandSelectedMovement(p, false, isDoubleClick);
+			//core->GetFirstSelectedPC(false)
+			Actor* selectedActor = core->GetFirstSelectedPC(false);
+			if (!selectedActor) {
+				//this could be a non-PC
+				selectedActor = core->GetGame()->selected[0];
+			}
+			if ((overContainer || target_mode == TARGET_MODE_PICK ) && selectedActor)
+			{
```

Comment:
The brace should be on the previous line

Reply:
why is `TARGET_MODE_PICK` being checked anyway? just to match originals? It was my understanding the desired behavior was to just never sent the whole party along to a container

Reply:
I modified it so only one character moves not only to the container but to pick up ground loot as well. From my perspective, it would suit gameplay. I haven't played this game for years, however, so i may be wrong. Should i cancel this addition?

Reply:
I dont think that answers why checking `TARGET_MODE_PICK` is desired

it might help to go read the discussion in the issue, but the door/infopoint stuff should probably be handled in a separate branch

Reply:
Oh, then it should definitely be removed. Loot piles are also containers, while the pick mode is for lock picking, so definitely not what you intended. Bashing and disarming are two other ways, but yeah, makes little sense to limit.

Reply:
done